spec/configuration_spec.rb

```diff
@@ -25,4 +27,9 @@
   it 'sets the candidates' do
     expect(SmartyStreets.configuration.candidates).to eq 1
   end
+
+  it 'sets the request timeout' do
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.